### PR TITLE
install in subfolder

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem
- *Root domain install only*

## Solution
- *Add the possibility to install in subfolder*

## Package_check results
---
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/codimd_ynh%20PR45%20(yalh76)/)  

### Buster result: 97672fd18d0422ab052c95222e5fdf59d97f8f6c
image

###Stretch result: 97672fd18d0422ab052c95222e5fdf59d97f8f6c
image

### CODE REVIEW

**Comment by ericgaspar** [comment]
Didn't realize this wasn't merged to master...

File: conf/nginx.conf [hunk 1]
```diff
@@ -1,25 +1,25 @@
-location ^~ / {
-  # Force usage of https
```
Comment (yalh76) [inline conf/nginx.conf:2]: We should keep the force usage of https

**Review by yalh76 — CHANGES_REQUESTED** [review]

**Review by yalh76 — APPROVED** [review]

**Review by lapineige — APPROVED** [review]

**Comment by yalh76** [comment]
Own CI results:

### Buster result: 97672fd18d0422ab052c95222e5fdf59d97f8f6c
image

###Stretch result: 97672fd18d0422ab052c95222e5fdf59d97f8f6c
image

**Comment by ericgaspar** [comment]
On merge alors!?

**Pull request merged**